tests/lib/schema_validation_test.py

```diff
@@ -62,13 +63,33 @@
                 'out-errors': 0,
             }
         }
+    ],
+    ROUTES: [
+        {
+            "table-name": "main",
+            "table-id": 254,
+            "origin": "static",
+            "metric": 100,
+            "destination": "0.0.0.0/0",
+            "next-hop-iface": "eth0",
```

Comment:
Unfortunately this shows a problem in our schema library - this does not match the schema but the test passes anyhow... This needs to be changed to `interface`.

Reply:
Fixed. Thanks.